hathor/indexes/manager.py

```diff
@@ -209,6 +225,22 @@ def add_tx(self, tx: BaseTransaction) -> bool:
 
         return r3
 
+    def _add_tx_post_consensus(self, tx: BaseTransaction) -> bool:
+        assert tx.hash is not None
+        tx_meta = tx.get_metadata()
+        assert tx_meta.validation.is_fully_connected()
+        if self.utxo:
+            self.utxo.add_tx(tx)
+
+        # XXX: this _should_ be here, but it breaks some tests, for now this is done explicitly in hathor.manager
+        # self.mempool_tips.update(tx)
```

Comment:
We are calling the index's method explicitly. Mostly because it kinds of falls in the same situation, I tried to just move it here, which would be the "more correct" place for it to be, but I had an issue with some tests, so I chose to not do this right now, but this is something we should do eventually.

Reply:
We were, and still are, calling it directly in `hathor/manager.py`, in several places.